freshmaker/errata.py

```diff
@@ -544,8 +555,36 @@ def has_jira_major_incidents(self, errata_id: str) -> bool:
         for issue in resp:
             if mi_pattern.search(issue["summary"]):
                 log.info(
-                    f"Found 'major incident' issue for advisory {errata_id}: {issue['key']} (errata_id={errata_id})"
+                    f"Found 'major incident' issue for advisory {errata_id}: {issue['key']}"
                 )
                 return True
 
         return False
+    
+    def has_jira_compliace_priority_label(self, errata_id) -> bool:
```

Comment:
```suggestion
    def has_compliance_priority_jira_issue(self, errata_id) -> bool:
```
Name suggestion and correct the typo `compliace`.